package.json

```diff
@@ -43,6 +43,7 @@
     "istanbul-coveralls": "^1.0.3",
     "mkdirp": "^0.5.1",
     "mocha": "^3.0.0",
+    "mocha-multi-reporters": "^1.1.7",
```

Comment:
I'd prefer not to add another devDep if possible

Reply:
That's probably fine. The advantage of this is that it allows us to use both the default reporter which prints live output and also creates a test results file. This allows us to produce the nicer test result reporting.

One workaround we could use here is to install this as a step in our ci process. The downside is that its a little less clean, we'd have to add that step and explicitly call `mocha --reporter mocha-multi-reporters --async-only` instead of just `npm test` in the testing step. We can also just not report results for that step.

Thoughts?

Reply:
I mentioned it above but it looks like mocha ships with an XUnit reporter, so I think we just need to make this a separate npm script and add another phase.  Would that make sense?

Reply:
I responded in the above thread, lets consolidate and continue conversation there 😄